index.html

```diff
@@ -953,8 +962,8 @@ <h4>createDisclosureData</h4>
             </li>
             <li>
 Return an object with properties matching |bbsProof|, "verifierLabelMap" for |labelMap|,
-|mandatoryIndexes|, |selectiveIndexes|, |revealDocument|, and |pseudonym|, if
-computed.
+|mandatoryIndexes|, |selectiveIndexes|, |revealDocument|, |pseudonym|, and, if
+computed, |lengthBBSMessages|.
```

Comment:
Is `|pseudonym|` now always computed?, or should rephrasing apply the `if computed` to both `|pseudonym|` and `|lengthBBSMessages|`?